src/applications/mhv/secure-messaging/sass/secure-messaging.scss

```diff
@@ -145,6 +152,33 @@
 
 .older-messages {
   margin-top: 24px;
+  margin-bottom: 24px;
+  padding-left: 5px;
+
+  a:visited {
+    color: $color-link-default;
+    text-decoration: underline;
+    transition-duration: 0.3s;
+    transition-timing-function: ease-in-out;
+    transition-property: color, background-color, border-color;
+  }
+  i {
+    margin: 0.5rem;
```

Comment:
Consider using a design system class instead. Same thing for these other selectors

Reply:
refactored where applicable. some designs we know for sure will change, so will keep that in mind for future edits